Casks/f/fastdmg.rb

```diff
@@ -0,0 +1,20 @@
+cask "fastdmg" do
+  version "1.0.2"
+  sha256 "405c99ee251ebe1a5f3e4cbec8b84e6211ee8b89e5abf0f50b7b2cee9567062a"
+
+  url "https://sveinbjorn.org/files/software/fastdmg/FastDMG-#{version}.zip"
+  name "FastDMG"
+  desc "Replacement for Apple's DiskImageMounter app"
```

Comment:
```suggestion
  desc "Alternative to Apple's DiskImageMounter app"
```

Minor nit, but "replacement" reads to me like Apple's option no longer exists.

Reply:
Can we write "Faster alternative to Apple's DiskImageMounter app"? As that's not like marketing fluffbut the whole reason this exist in the first place.

Reply:
We tend not to because statements like "faster" can be a bit circumstantial and may not stay true forever.

Reply:
Ok, Nit commited, now need a new approver :(

Reply:
I'm still here, thanks @Moulick

Reply:
Hah. I assumed that if I took your code suggestion and as a commit now shows up as co-authored by me + you, GitHub would not accept your approval. But maybe `homebrew-cask` has it allowed in the branch ruleset(s)?